packages/performance-tests/package.json

```diff
@@ -41,6 +41,7 @@
     "@types/node": "14.14.31",
     "@types/yargs": "^12.0.5",
     "chai": "^4.3.6",
+    "encoding": "^0.1.13",
```

Comment:
don't introduce dependencies we don't use... if you're making a monorepo on top of this repo, alter its hoisting settings to access this.